rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -183,6 +189,10 @@ class PlayerImpl
   /// \return Hashtable representing topic to publisher map excluding inner clock_publisher
   std::unordered_map<std::string, std::shared_ptr<rclcpp::GenericPublisher>> get_publishers();
 
+  /// \brief Getter for clients corresponding to services
+  /// \return Hashtable representing service name to client map
+  std::unordered_map<std::string, std::shared_ptr<rclcpp::GenericClient>> get_clients();
```

Comment:
Comments from MichaelOrlov

get_services_clients() would be more clear. Please rename.

Reply:
Updated